src/Features/Core/Portable/PdbSourceDocument/IPdbSourceDocumentLogger.cs

```diff
@@ -0,0 +1,16 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace Microsoft.CodeAnalysis.PdbSourceDocument
+{
+    internal interface IPdbSourceDocumentLogger
```

Comment:
It's not clear to me why this was added... why not just `ILogger`?

Reply:
These logs are specifically for trying to help library authors, if source link isn't configured properly or something, and their symbols aren't right or something. They go to the output window (screenshots in https://github.com/dotnet/roslyn/pull/57996). Our normal logs are visible easily, and we wouldn't want all of the extra noise from all of the other things that ILogger gets told about

Reply:
A comment would be nice to help explain that :)